File: Herencia/ejercicio 1/Persona.java
package herencia;
public class Persona {
    protected String nombre;
    protected String apellido;
    protected String ci;

    public Persona(String nombre, String apellido, String ci) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.ci = ci;
    }

    public String mostrarDatos() {
        return nombre + " " + apellido + " - CI: " + ci;
    }
}

File: Herencia/ejercicio 1/persona.py
class Persona:
    def _init_(self, nombre: str, apellido: str, ci: str):
        self.nombre = nombre
        self.apellido = apellido
        self.ci = ci

    def mostrar_datos(self) -> str:
        return f"{self.nombre} {self.apellido} - CI: {self.ci}"